web/backend/src/Server.ts

```diff
@@ -26,6 +26,27 @@ const app = express();
 
 app.use(morgan('tiny'));
 
+let enf: Enforcer;
+
+const enforcerLoading = newEnforcer('model.conf', 'policy.csv');
+const e = newEnforcer('model.conf', 'policy.csv');
+const port = process.env.PORT || 5000;
+
+Promise.all([enforcerLoading])
+  .then((res) => {
+    [enf] = res;
```

Comment:
Do you need a `Promise.all` if you're looking at just one promise resolving?
Why can't you do the following ? 
```suggestion
enforcerLoading
  .then((enf) => {
```
I may be missing something here, so do let me know if I'm wrong 😁